contrib/html/html.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import defineFunction, {ordargument} from "../../src/defineFunction";
+import buildCommon from "../../src/buildCommon";
+import mathMLTree from "../../src/mathMLTree";
+import domTree from "../../src/domTree";
+
+import {buildExpression as htmlBuildExpression} from "../../src/buildHTML";
+import {buildExpression as mmlBuildExpression} from "../../src/buildMathML";
+
+defineFunction({
```

Comment:
It feels a little odd that some of this logic lies outside of the source (like this), but other logic needs to lie within the source (like Parser/types). That being said, I'm not sure there's a better way to implement it at this time without a major change.